config/main.py

```diff
@@ -681,11 +681,16 @@ def _stop_services():
 
 def _get_sonic_services():
     out = clicommon.run_command("systemctl list-dependencies --plain sonic.target | sed '1d'", return_cmd=True)
-    return [unit.strip() for unit in out.splitlines()]
+    return (unit.strip() for unit in out.splitlines())
```

Comment:
>) [](http://example.com/codeflow?start=53&length=1)

Is there any problem when using list instead of tuple?

Reply:
It is a generator. I suppose generator is more efficiency.

```python
a = (str(x) for x in range(10))
print(type(a))
```

Output:
```
<class 'generator'>
```